web/src/components/navbar/desktop/NavBarDesktopHomeLogo.tsx

```diff
@@ -0,0 +1,26 @@
+import { NavBarDashboardLink } from "@/components/navbar/NavBarDashboardLink";
+import { NavBarVerticalLine } from "@/components/navbar/NavBarVerticalLine";
+import { NavbarBusinessNjGovLogo } from "@/components/navbar/NavbarBusinessNjGovLogo";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { ReactElement } from "react";
+
+interface Props {
+  previousBusinessId: string | undefined;
```

Comment:
pebble - spencer had a great reason for using `?` instead of explicitly using undefined as a type. I forgot why, but we can simply say `previousBusinessId?: string`

Reply:
Sure